src/components/variables.scss

```diff
@@ -5,3 +5,4 @@ $s-height: 24px;
 $m-height: 28px;
 $l-height: 36px;
 $xl-height: 44px;
+$border-width: 1px;
```

Comment:
We have different border widths in our component. It's not right to put it in common variables